### PR TITLE
Add RELION v3.1.4 and RELION v5.0-beta built with `foss2022a-CUDA-11.7.0`

### PR DESCRIPTION
**Pull request by Neves-P**
At the moment this is still a draft as the `himem` nodes are busy so I haven't ingested the tarballs yet. Once the tarballs are ready, I'll ingest and undraft (and amend my sin of accidentally pushing this to master earlier, shame shame)

### CODE REVIEW

**Review by camarocico — COMMENTED** [review]
This looks good to me, hopefully it installs well on the `himem` nodes, then we can merge.

File: easyconfigs/r/RELION/RELION-5.0-beta-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),    
+    ('X11', '20220504'),
+    ('FLTK', '1.3.8'),
+    ('LibTIFF', '4.3.0'),
+]
+
+# lowest supported capability is 3.5
+cuda_compute_capabilities = ['3.5', '3.7', '5.0', '5.2', '5.3', '6.0', '6.1', '6.2', '7.0']
```
Comment (bedroge) [inline easyconfigs/r/RELION/RELION-5.0-beta-foss-2022a-CUDA-11.7.0.eb]: The A100 has compute capability 8.0, so may be worth adding that one as well (if supported by Relion).

Comment (Neves-P) [reply easyconfigs/r/RELION/RELION-5.0-beta-foss-2022a-CUDA-11.7.0.eb]: Well spotted :) The manual simply says 3.5+, so I'll add 8.0 and rebuild

**Comment by Neves-P** [comment]
I rebuilt all the tarballs with 8.0 in `cuda_capabilities` and ingested everything 👍

**Review by bedroge — APPROVED** [review]
It would be nice if the easyconfig could take the CUDA capability from the EB config, but I think I've tried to do something like that for a similar easyconfig in the past, and it didn't really work (guess it has to be done in an easyblock). So let's just merge this for now.

**Pull request merged**